pom.xml

```diff
@@ -5,7 +5,7 @@
     <groupId>com.seeq.jscience</groupId>
     <artifactId>jscience</artifactId>
     <packaging>jar</packaging>
-    <version>5.0.0</version>
+    <version>5.4.2-SNAPSHOT</version>
```

Comment:
You shouldn't need to increment the version to get changes to propagate since it's a snapshot. Consider asking about the best practice for this in the gradle slack channel.

Reply:
Yeah, this was silly. Following up in #gradle.